vcbuild.bat

```diff
@@ -144,7 +145,33 @@ if defined noprojgen if defined nobuild if not defined sign if not defined msi g
 
 @rem Set environment for msbuild
 
+set msvs_host_arch=x86
```

Comment:
It actually should be added to the VS2015 case as well, but that's for a different PR